### PR TITLE
message

### PR DESCRIPTION
**Pull request by dev-raj-kumar**

### CODE REVIEW

File: app/src/main/res/layout/activity_splash.xml [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/splash_image_view"
+        android:src="@drawable/splash_screen_image"
+        android:scaleType="centerCrop"
+        android:layout_marginTop="50dp" />
+    <TextView
+        android:layout_width="90dp"
+        android:layout_height="16dp"
```
Comment (krharsh17) [inline app/src/main/res/layout/activity_splash.xml]: Remove this redundant element.

File: app/src/main/res/layout/activity_splash.xml [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
+        android:layout_height="wrap_content"
+
+        android:layout_below="@id/splash_image_view"
+        android:layout_marginLeft="86dp"
+        android:layout_marginTop="40dp"
+        android:fontFamily="@font/ubuntu_medium"
+        android:text="@string/app_upper_name"
+        android:textColor="#FFFFFF"
+        android:textSize="48dp"
+        android:textStyle="normal|bold" />
```
Comment (krharsh17) [inline app/src/main/res/layout/activity_splash.xml]: Use one TextView to display 'tinyhouse salesforce' on the splash page. To move a word in the next line, use the next line escape sequence ('\n'). Merge these two TextViews into one TextView.

File: app/src/main/res/layout/activity_sign_up.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (krharsh17) [inline app/src/main/res/layout/activity_sign_up.xml]: The white toolbar from the design is missing. Add a toolbar.

File: app/src/main/res/layout/activity_splash.xml (hunk 1 above)
Comment (dev-raj-kumar) [reply app/src/main/res/layout/activity_splash.xml]: It is to hide the mark of the designer.
If it be removed then the height if the image would have a fixed sone numeric value

File: app/src/main/res/layout/activity_signup.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
```
Comment (krharsh17) [inline app/src/main/res/layout/activity_signup.xml]: Toolbar still hasn't been added. Check the designs.

**Pull request merged**